### PR TITLE
Effnet docs release

### PR DESCRIPTION
**Pull request by AlexEMG**
**This is wip to update the docs & release 2.1.10.**

_This is also benchmarked. My results were (Run with DLC *2.1.9* in Jan 2021) for 100 k iterations._

**ResNet50
Imgaug:
Results for 100000  training iterations: 95 1 train error: 2.13 pixels. Test error: 2.22  pixels.
With pcutoff of 0.4  train error: 2.13 pixels. Test error: 2.22 pixels

Scalecrop:
Results for 100000  training iterations: 95 2 train error: 1.47 pixels. Test error: 1.77  pixels.
With pcutoff of 0.4  train error: 1.47 pixels. Test error: 1.77 pixels

Tensorpack:
Results for 100000  training iterations: 95 3 train error: 2.09 pixels. Test error: 2.36  pixels.
With pcutoff of 0.4  train error: 2.09 pixels. Test error: 2.36 pixels

**EffNet-b3
Imgaug:
Results for 100000  training iterations: 95 4 train error: 2.39 pixels. Test error: 2.57  pixels.
With pcutoff of 0.4  train error: 2.39 pixels. Test error: 2.57 pixels

Scalecrop:
Results for 100000  training iterations: 95 5 train error: 2.26 pixels. Test error: 2.24  pixels.
With pcutoff of 0.4  train error: 2.26 pixels. Test error: 2.24 pixels

Tensorpack:
Results for 100000  training iterations: 95 6 train error: 1.65 pixels. Test error: 2.24  pixels.
With pcutoff of 0.4  train error: 1.65 pixels. Test error: 2.24 pixels

****NOTE: for the EfficientNet we have not optimized the learning rate schedule for batch size 1 and large images. Here we showed rigorously for different datasets and models that EffNets outperform ResNets and MobNets as expected from the imagenet performance: https://openaccess.thecvf.com/content/WACV2021/html/Mathis_Pretraining_Boosts_Out-of-Domain_Robustness_for_Pose_Estimation_WACV_2021_paper.html

(thereby we cross-validated the learning rate schedule for each model, you might want to do this as well on your data!).****

### CODE REVIEW

**Review by MMathisLab — APPROVED** [review]

**Comment by jeylau** [comment]
@AlexEMG, why rolling back predict_.py?

**Comment by AlexEMG** [comment]
Jan 2020:
MobileNetV2 0.35
Results for 10000  training iterations: 95 1 train error: 5.79 pixels. Test error: 5.63  pixels.
With pcutoff of 0.4  train error: 5.79 pixels. Test error: 5.63 pixels

ResNet 50
Results for 10000  training iterations: 95 2 train error: 3.61 pixels. Test error: 3.7  pixels.
With pcutoff of 0.4  train error: 3.61 pixels. Test error: 3.7 pixels

EffNet-b3
Results for 10000  training iterations: 95 3 train error: 6.86 pixels. Test error: 6.63  pixels.
With pcutoff of 0.4  train error: 6.86 pixels. Test error: 6.63 pixels

**Comment by MMathisLab** [comment]
@AlexEMG it should be 2.1.10, not 2.1.9.10^
